egs/multi_en/s5/local/dict.patch

```diff
@@ -0,0 +1,61 @@
+1,12d0
```

Comment:
It's used in `local/prepare_dict.sh`. There's a comment right above when it's used:
```bash
$ grep -B 1 'dict.patch' local/prepare_dict.sh 
# Get rid of punctuation words
patch -o $dir/lexicon2.txt $dir/lexicon1.txt local/dict.patch
```